src/module/number_guess.py

```
@@ -0,0 +1,113 @@
+class GuessInfo:
+    status: GuessStatus
+    target: int
+    bound: tuple[int, int]
+    trials: int
+
+
+_guess_info = {}
+_guess_info_lock = threading.Lock()
+
+
+def start_game(message: RobotMessage):
```

Pylint (3.3.7)

[convention] Missing function or method docstring

(C0116)